### PR TITLE
simplify the header of the benchmark table

### PR DESCRIPTION
**Pull request by lvhan028**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: benchmark/profile_generation.py [hunk 1]
```diff
@@ -381,20 +381,26 @@ def main():
...
-                '1st_token_latency(min)(s)', '1st_token_latency(max)(s)',
-                '1st_token_latency(ave)(s)', 'percentile50(s)',
-                'percentile75(s)', 'percentile95(s)', 'percentile99(s)',
-                'throughput(token/s)', 'mem_per_proc(GB)', 'mem_per_gpu(GB)'
+                'batch',
+                'prompt_tokens',
+                'completion_tokens',
+                'throughput(out tok/s)',
+                'mem(GB)'
+                'FTL(ave)(s)',
```
Comment (AllentDan) [inline benchmark/profile_generation.py:389]: There exists both `FTL(ave)` and `first_token_latency_ave` in our scripts.

**Review by AllentDan — COMMENTED** [review]
Please fix lint error.

**Comment by lvhan028** [comment]
> Please fix lint error.

The linting error is:
```
FILE: ./README.md
[✖] https://huggingface.co/internlm/internlm-chat-7b-v1_1 → Status: 404
```
It is not our issue

**Review by AllentDan — APPROVED** [review]
LGTM

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**